guide/src/building_and_distribution.md

```diff
@@ -82,6 +82,36 @@ You can then open a Python shell in the output directory and you'll be able to r
 
 See, as an example, Bazel rules to build PyO3 on Linux at https://github.com/TheButlah/rules_pyo3.
 
+#### macOS
+
+On macOS, because the `extension-module` feature disables linking to `libpython` ([see the next section](#the-extension-module-feature)), some additional linker arguments need to be set. `maturin` and `setuptools-rust` both pass these arguments for automatically, but projects using manual builds will need to set these directly in order to support macOS.
```

Comment:
```suggestion
On macOS, because the `extension-module` feature disables linking to `libpython` ([see the next section](#the-extension-module-feature)), some additional linker arguments need to be set. `maturin` and `setuptools-rust` both pass these arguments for PyO3 automatically, but projects using manual builds will need to set these directly in order to support macOS.
```